src/renderers/webxr/WebXRManager.js

```diff
@@ -199,18 +202,47 @@ function WebXRManager( renderer, gl ) {
 
 			}
 
-			const layerInit = {
-				antialias: attributes.antialias,
-				alpha: attributes.alpha,
-				depth: attributes.depth,
-				stencil: attributes.stencil,
-				framebufferScaleFactor: framebufferScaleFactor
-			};
+			if ( session.renderState.layers === undefined ) {
```

Comment:
Are we going to have to maintain two code paths from now on?
Or is this transitional and we'll able to just use the layers code path in the future?

Reply:
I know Microsoft is going to start on layers and Google also said that they're interested because it gives them access to texture arrays and in the near future WebGPU.
Once they land support, we can remove the dual path.